lua/lv-lightspeed/init.lua

```diff
@@ -0,0 +1,20 @@
+local M = {}
+
+M.config = function()
+	require("lightspeed").setup({
+		jump_to_first_match = true,
+		jump_on_partial_input_safety_timeout = 400,
+		highlight_unique_chars = true,
```

Comment:
Author here: `highlight_unique_chars` is unfortunately _extremely_ slow if the window area is big (the bottleneck seems to be the native `search()` itself, cannot really speed it up), so I deliberately turned it off by default, and I would recommend doing so here.

Reply:
Thanks for the suggestion.